Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Control.cs

```diff
@@ -42,7 +52,27 @@ public float getValue()
             return value;
         }
 
-        public virtual void Start()
+        /// <summary>
+        /// Returns the current value mapped onto a range between 0% and 100%.
+        /// </summary>
+        /// <returns>Normalized Value</returns>
+        public float getNormalizedValue()
+        {
+            return Mathf.Abs(Mathf.Round((value - cvr.controlMin) / (cvr.controlMax - cvr.controlMin) * 100));
+        }
+
+        public void SetContent(GameObject content, bool hideContent)
```

Comment:
Right I've created a draft Documentation:

The whole file is at this gist ->https://gist.github.com/thestonefox/48f9d887cfcf5b804a70f0dddad3e7ea

It includes your doc changes as well so you should just be able to get the raw file from the gist and overwrite your local one and repush the PR.